addons/cargo/stringtable.xml

```diff
@@ -329,6 +333,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_paradropTimeCoefficent">
             <English>Paradrop Time Coffecient</English>
+            <German>Türlast Zeitkoeffizient</German>
```

Comment:
"Türlast"? Never heard of that and no google results. But I see that's already used in `STR_ACE_Cargo_paradropButton`

Reply:
Ausdruck wird in der BW verwendet.

Reply:
Das is entschieden länger als der längste Text in der englischen Übersetzung (welcher da wäre der erste Eintrag in Nametags): ![turlastabsetzungsdaueranpassung](https://user-images.githubusercontent.com/37950828/43101873-c97222dc-8ec9-11e8-8283-eba7253accef.png)
Glaub "Türlastabsetzungsdaueranpassung" wär eh besseres Deutsch.


Und `<English>Paradrop Time Coffecient</English>` müsste `<English>Paradrop Time Coefficient</English>` sein.

Reply:
The correct translation for "paradrop" is "Fallschirmabwurf" and not "Türlast".
Terms used in the Bundeswehr are very bad for a translation like this. They're not common use in language. This is a German translation and not a Bundeswehrdeutsch translation.

Reply:
"Fallschirmabwurf-Zeitkoeffizient"

Reply:
Im ok with Fallschirmabwurf.

@dedman my google find thousands of results for Türlast in conection with Flugzeug ^^

Reply:
dunno who you mentioned there but not me :D 
Okey yeah with Flugzeug I have 51 results. Including one where a guy talks about ACE and that  `Türlast` is a bad translation :D

Reply:
"Tuerlast" is the right term for dropped cargo out of a plane for the german army. We used this term when we did paradrops and our heavy equipment needed to be dropped separately.

Reply:
>"Tuerlast" is the right term for dropped cargo out of a plane for the german army.
See: https://github.com/acemod/ACE3/pull/6466#discussion_r204638133

Reply:
The english translation uses army terms as well. So where is the problem doing the same in german?

Reply:
The English version of this string is: "Paradrop Time Coffecient", which is, while very technical, standard English. If there is a bad (=not day to day English) translation anywhere, then it should be changed as well.

>This is a military simulation after all.

And it is also a video game used by civilians. A tooltip is there to be understood by everyone.

Reply:
@commy2 I disagree

Reply:
I'd be fine with the term being used, and defined in an extra sentence, even if that were a feature of the German string only. Tooltips can be quite lengthy and still fit. Aside from that, I rest my case.